Doc/faq/programming.rst

```diff
@@ -518,53 +518,57 @@ desired effect in a number of ways.
 
 1) By returning a tuple of the results::
 
-      def func2(a, b):
-          a = 'new-value'        # a and b are local names
-          b = b + 1              # assigned to new objects
-          return a, b            # return new values
-
-      x, y = 'old-value', 99
-      x, y = func2(x, y)
-      print(x, y)                # output: new-value 100
+      >>> def func2(a, b):
```

Comment:
Funny that we have func2 before func1.  Switch them?

Reply:
Fix